blockly-core/core/ui/fields/field_dropdown.js

```diff
@@ -133,7 +133,9 @@ Blockly.FieldDropdown.prototype.showEditor_ = function() {
   menuDom.style.borderColor = 'hsla(' + this.sourceBlock_.getColour() + ', ' +
     this.sourceBlock_.getSaturation() * 100 + '%, ' +
     this.sourceBlock_.getValue() * 100 + '%' + ', 0.5)';
-
+  menuDom.style.overflow = "scroll";
```

Comment:
I recommend
```javascript
menuDom.style.overflowY = "auto";
```
`overflowY` so that you don't get the horizontal scrollbar, `auto` which should make the scrollbar only appear if it's needed - if the content exceeds the element height.